.github/workflows/build-doxygen.yml

```diff
@@ -44,7 +44,7 @@ jobs:
 
     - name: Assign Doxygen PR reviewer
       run: |
-        PR_REVIEWER="mirenradia" && [[ $GITHUB_ACTOR == "mirenradia" ]] && PR_REVIEWER="KAClough"
+        [[ $GITHUB_ACTOR == "KAClough" ]] && PR_REVIEWER="SamuelBrady"
```

Comment:
Do you want to set @KAClough as the `PR_REVIEWER` in the case she's not the `GITHUB_ACTOR`?

Reply:
I can't see how anyone else can end up as the GITHUB_ACTOR, can it get changed somehow? Also is there a benefit to the reviewer changing?

Reply:
So basically we want to achieve 2 things.
1. Remove me as a potential `PR_REVIEWER`.
2. Assign the `PR_REVIEWER` such that it is always different to the automated PR author since the reviewer of a PR can't be the same as the PR.

Looking at the [`peter-evans/create-pull-request` action documentation](https://github.com/peter-evans/create-pull-request#action-inputs), it seems to suggest that the author of the automated PR should be the GitHub actor (i.e. the user that triggered the workflow). Having said that, looking at the [history of these automated PRs](https://github.com/GRChombo/GRChombo.github.io/pulls?q=is%3Apr+in%3Atitle+Update+Doxygen), the author always seems to have been me :thinking:.

Maybe the solution is to explicitly assign the author. Ideally, we'd assign a GitHub bot as the author but that sounds like too much effort to be worth it...

Reply:
I was intending to just set GITHUB_ACTOR to KAClough (but forgot to change the first part), would it not work to set it there?

Reply:
I don't think you should change the `GITHUB_ACTOR`. Best to set the `author` of the automated PR explicitly.